src/index.html

```diff
@@ -11,7 +11,152 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      >
+      <fieldset class="pesonal_information">
+        <legend>Personal information</legend>
+        <label class="pesonal_information-input">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          required
+          >
+        </label>
+        <label class="pesonal_information-input">
+          Name:
+          <input
+          type="text"
+          name="name"
+          autocomplete="off"
+          required
+          >
+        </label>
+        <label class="pesonal_information-input">
+          How old are You?
+          <input type="number"
+          name="age"
+          min="1"
+          max="100"
+          value="12"
+          required
+          >
+        </label>
+        <label class="pesonal_information-input" required>
+          Full date of birth:
+          <input type="date" name="date of birth">
+        </label>
+        <label>
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="terms"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="registration">
+        <legend>Registration</legend>
+        <label class="registration_input">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          autocomplete="off"
+          required
+          >
+        </label>
+        <label>
+          Password:
+          <input
+          type="password"
+          name="password"
+          autocomplete="off"
+          required
+          minlength="5"
+          maxlength="20"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="facts">
+        <legend>An interesting fact about you!</legend>
+        <label>
```

Comment:
move this label only to the input

it chooses the first option when I click on `Do you love cats?`

Reply:
<img width="177" alt="image" src="https://user-images.githubusercontent.com/102411130/231080112-612d4c86-b00f-4a36-a01f-7a6ef295ad18.png">